torchci/components/TorchAgentPage.tsx

```diff
@@ -758,8 +758,43 @@ export const McpQueryPage = () => {
       )}
 
       <Typography variant="h4" gutterBottom>
-        PyTorch Grafana Agent
+        TorchAgent
       </Typography>
+
+      <Typography
+        variant="body1"
+        paragraph
+        sx={{
+          mb: 3,
+          p: 2,
+          backgroundColor: "background.paper",
+          borderRadius: 1,
+          border: "1px solid",
+          borderColor: "divider",
+        }}
+      >
+        Welcome to TorchAgent, your intelligent assistant for PyTorch
+        infrastructure analysis and monitoring. This tool helps you create
+        custom time-series visualizations, analyze CI/CD metrics, and gain
+        insights into the PyTorch development workflow. Simply describe what
+        you&apos;d like to explore, and TorchAgent will generate the appropriate
+        queries and dashboards for you. Data we have access to:
+        <ul>
+          <li>
+            PyTorch GitHub repository data (comments, issues, PRs, including
+            text inside of these)
+          </li>
+          <li>
+            PyTorch GitHub Actions CI data (build/test/workflow results, logs,
+            artifacts)
```

Comment:
do we actually have logs and artifacts in ch?

Reply:
log classifications, changed this line to reflect that